### ISSUE TITLE
2-Digit Country Code errored out

### ISSUE THREAD

**Issue by UtahDave** [Daplie/daplie-tools#5]
During the purchase process I was prompted to enter my 2-Digit Country Code.
I wasn't sure what to put there so I entered `01`.

This was apparently the wrong answer. :) I then received the following output:

```
2-Digit Country Code: { id: 'b89c6b3c183e12ef4dc826377189642e',
  type: 'tel',
  node: '+1801XXXXXXX',
  createdAt: 1459979543748,
  updatedAt: 1459979543748 }
{ firstName: '<my name>',
  lastName: '<my last name>',
  streetAddress: '<my address>',
  extendedAddress: '',
  locality: '<my city>',
  region: '<my state>',
  postalCode: '<my zip>',
  countryCode: '01',
  id: '1542f4d9f03dd8e72d997fa360f242b7' }
{ id: '5f2a3cde66bed56ac13783908010022f',
  type: 'email',
  node: '<my email>',
  createdAt: 1459979543910,
  updatedAt: 1459979543910 }
Possibly Unhandled Rejection at:
Promise:  { _bitField: 152305664,
  _fulfillmentHandler0: 
   { name: 'TypeError',
     message: 'Cannot read property \'id\' of null' },
  _rejectionHandler0: undefined,
  _promise0: undefined,
  _receiver0: undefined }
undefined
Reason:  { name: 'TypeError',
  message: 'Cannot read property \'id\' of null' }
undefined
```

**Comment by coolaj86** [comment]
I'm about halfway done fixing this so that it will say "Country :" instead of "2-digit Country Code" and it will also autocomplete no matter whether you type the 2-character code, 3-character code, or the full name (Democratic Republic of Congo) or normal name (Congo) and it won't allow you to enter in an incorrect value at all anyway (just like the domain extensions).

**Comment by coolaj86** [comment]
Very fixed in `v1.0.0-alpha.905`. Simply `npm install -g daplie-tools` to reinstall and see what you think.

**Comment by brysonchill** [comment]
This is really smooth. Well done!
